### ISSUE TITLE
feature request: offline capable

### ISSUE THREAD

**Issue by tjiahen** [Nedomas/databound#5]
Any plan to make databound offline capable with Service Worker and indexedDB?

Maybe it is a better idea to create another repo for offline capability.

**Comment by Nedomas** [comment]
Hello @wizztjh 

Thanks for raising this :heart: :heart: :heart:

I never needed this myself, but if we can collaborate on the solution to this.

Could you setup an app and use your fork of databound so we could start working on this?

**Comment by jpalumickas** [comment]
I would agree with @Nedomas to start this idea on the new project integrated with databound, not on this one.
